server.go

```diff
@@ -99,7 +100,9 @@ func (users *Users) UpdateUserInfo(anUser *User) bool {
 	defer users.UserListMutex.Unlock()
 	users.UserList[anUser.Email].Update(anUser)
 	return true
+	// Comment
 }
+// Comment
```

Comment:
Should we return some errors for user not being updated?

Reply:
There are two options - user is there (email is known) so update could be done, user is not there so the update cannot be done. To indicate those two states I'm using bool.
Currently there is no check whether the newly provided data differs from what is already in there. As "new feature" this cloud be provided.
Also I do not check whether the user that is updated is the one calling the API (the http credentials). This should be considered as bug.